consensus/state.go

```diff
@@ -521,6 +521,13 @@ func (cs *ConsensusState) updateToState(state *sm.State) {
 		lastPrecommits = cs.Votes.Precommits(cs.CommitRound)
 	}
 
+	// Log whether this node is a validator
+	if privVal := cs.privValidator; privVal != nil && validators.HasAddress(privVal.GetAddress()) {
```

Comment:
> Actually, do we log whether we are the proposer (if not we should probably add that)?

Great point! So let's just do the logging there: https://github.com/tendermint/tendermint/blob/master/consensus/state.go#L809

Either you're a proposer, or you're not a proposer, or you're not a validator. I'd think that, if you're a validator, it should log whether you're the proposer or not, otherwise nothing. But I guess we could have three cases and log for each of them: proposer, validator but not proposer, not a validator.